Google.Api.Generator.Tests/ProtoTests/Paginated/Testing.Paginated.StandaloneSnippets/PaginatedClient.ResourcedMethod2Snippet.g.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Testing.Paginated.Snippets
 {
+    // [START paginated_generated_Paginated_ResourcedMethod_sync_flattened2]
```

Comment:
Where we've got disambiguation like this, does it matter if the index isn't stable over time?

Reply:
Unfortunatelly yes, it should be stable.
If we get all systems to depend on metadata only, then we won't care about the region tag that much, but that's not the case right now (although I'm trying).
That said, in reality (and this applies for all languages), these samples will most likely be included in reference docs only, either via CloudRad or API reference docs, and none of that will depend on the region tag.
I don't think these samples will be broadly consumed by TWs, which is what the region tag is ultimately useful for.